goat/locale/goat.csv

```diff
@@ -1,40 +1,41 @@
-"Use","Use","Benutz","Użyj"
-"Use/pick up","Use/pick up","Verwenden/abholen","Użyj/podnieś"
-"WSAD,Arrow keys","WSAD\nArrow keys","WSAD\nPfeiltasten","WSAD\nKlawisze strzałek"
+keys,en,de,pl,fr
+Ambient occlusion,Ambient occlusion,Umgebungsverdeckung,Ambient occlusion,Occlusion ambiante
+Back,Back,Zurück,Wróć,Retour
+Camera rotation,Camera rotation,Kameradrehung,Obrót kamery,Rotation de la caméra
+Cancel,Cancel,Absagen,Anuluj,Annuler
+Controls,Controls,Steuerung,Sterowanie,Contrôles
+Display,Display,Anzeige,Obraz,Affichage
+Effects,Effects,Effekten,Efekty,Effets
+English,English,Englisch,Angielski,Anglais
+Esc,Esc,Esc,Esc,Échap.
+Exit,Exit,Verlassen,Wyjdź,Sortir
+French,French,Französisch,Francuski,Français
+Fullscreen,Fullscreen,Vollbildschirm,Pełen ekran,Plein écran
+German,German,Deutsch,Niemiecki,Allemand
+Glow,Glow,Glühen,Poświata,Éclat
+LMB,LMB,LMT,LPM,Clic gauche
+Language,Language,Sprache,Język,Langue
+Mouse sensitivity (3D),Mouse sensitivity (3D),Mausempfindlichkeit (3D),Czułość myszy (3D),Sensibilité de la souris (3D)
+Mouse,Mouse,Maus,Mysz,Souris
+Movement,Movement,Bewegung,Ruch,Mouvement
+Music,Music,Musik,Muzyka,Musique
+Open context inventory,Open context inventory,Kontextinventar öffnen,Otwórz ekwipunek kontekstowy,Ouvrir l’inventaire contextuel
+Open inventory,Open inventory,Inventar öffnen,Otwórz ekwipunek,Ouvrir l’inventaire
+Polish,Polish,Polnisch,Polski,Polonais
+Press RMB to rotate. Drag and drop other items to use them on this one.,Press RMB to rotate. Drag and drop other items to use them on this one.,"Drücken Sie RMT, um sich zu drehen. Ziehen Sie andere Gegenstände per Drag und Drop, um sie für dieses zu verwenden.","Naciśnij PPM, aby obrócić. Przeciągnij i upuść inne przedmioty, aby użyć ich na tym.",Tenez clic droit appuyé pour tourner. Glissez-déposez d’autres objets pour les utiliser sur celui-ci.
+RMB,RMB,RMT,PPM,Clic droit
+Reflections,Reflections,Reflexionen,Odbicia,Réflexions
+Reset rotation,Reset rotation,Drehung zurücksetzen,Resetuj obrót,Réinitialiser la rotation
+Resume,Resume,Fortsetzen,Wznów,Reprendre
+Scope,Scope,Sucher,Celownik,Contexte
+Settings,Settings,Einstellung,Ustawienia,Paramètres
+Shadows,Shadows,Schatten,Cienie,Ombres
+Show settings screen,Show settings screen,Einstellungsbildschirm anzeigen,Pokaż ekran ustawień,Affichager l’écran des paramètres
+Sound,Sound,Schall,Dźwięk,Son
+Subtitles,Subtitles,Untertitel,Napisy,Sous-titres
+Tab,Tab,Tab,Tab,Tabulation
+Take a screenshot,Take a screenshot,Einen Screenshot machen,Wykonaj zrzut ekranu,Prendre une capture d’écran
+There are no items in the inventory,There are no items in the inventory,Es gibt keine Gegenstände im Inventar,W ekwipunku nie ma żadnych przedmiotów,Il n’y a pas d’objets dans l’inventaire
+Use,Use,Benutz,Użyj,Utiliser
+Use/pick up,Use/pick up,Verwenden/abholen,Użyj/podnieś,Utiliser/prendre
+"WSAD,Arrow keys",WSAD\nArrow keys,WSAD\nPfeiltasten,WSAD\nKlawisze strzałek,ZQSD\nTouches directionnelles
```

Comment:
This one seems a bit too long, and it breaks the layout of the whole column. Can you find a shorter version (it doesn't have to be a literal translation, but something with a similar meaning)? If not, the layout can be rewritten here (e.g. the width of the column could be increased), but we need to keep in mind other translations. If you have an idea how to fix it, please let me know ;)

![Screenshot from 2020-07-18 22-14-53](https://user-images.githubusercontent.com/36821133/87863374-b2aad080-c95a-11ea-9c60-7813c49329a5.png)